pkgs/applications/virtualization/docker/default.nix

```diff
@@ -64,16 +59,71 @@ rec {
 
       NIX_CFLAGS_COMPILE = "-DMINIMAL=ON";
     });
+
+    moby = buildGoPackage ((optionalAttrs (stdenv.isLinux)) rec {
+      name = "moby-${version}";
+      inherit version;
+      inherit docker-runc docker-containerd docker-proxy docker-tini;
+
+      src = fetchFromGitHub {
+        owner = "moby";
+        repo = "moby";
+        rev = mobyRev;
+        sha256 = mobySha256;
+      };
+
+      goPackagePath = "github.com/docker/docker";
+
+      nativeBuildInputs = [ makeWrapper pkg-config go-md2man go libtool installShellFiles ];
```

Comment:
Please change pkgconfig in line two to fix the eval error.

Reply:
Done.